plugins/bigquery/dbt/adapters/bigquery/impl.py

```diff
@@ -346,6 +346,36 @@ def execute_model(self, model, materialization, sql_override=None,
 
         return res
 
+    @available.parse_none
```

Comment:
I'm going to be pedantic about types here: This should probably be `@available.parse(lambda *a, **k: True)` (or `False`).